parsers/round-number/README.md

```diff
@@ -0,0 +1,348 @@
+# Round number
+
+## Description
+
+Rounds any measurement with specific precision. 
+You can easily round any measurement value (font size, line height, blur etc.) and specify the number of decimals.
```

Comment:
You can easily round any measurement value (font size, line height, blur etc.) and specify the number of decimals.
By specifying a mode, you can control the round function.

⬇️

Some measurement token values like a font size, a line height or a shadow blur may need to be rounded.
By specifying a `mode`, you can control the round function more precisely.